src/sdk/analytics.js

```diff
@@ -1,8 +1,9 @@
 import {
   HiiveAnalytics,
   HiiveEvent,
-} from "@newfold-labs/js-utility-ui-analytics";
+} from "@newfold/js-utility-ui-analytics";
 import { NewfoldRuntime } from "@newfold/wp-module-runtime";
+import data from "@wordpress/data";
```

Comment:
wondering if we should change this to `import { * as wpData } from "@wordpress/data"` ?